src/materialized/src/http/templates/flamegraph.html

```diff
@@ -70,10 +70,28 @@ <h1>{{ title }}</h1>
 </div>
 
 <script>
+    function toHumanBytes(n) {
+        const tokens = ['B', 'KiB', 'MiB', 'GiB', 'TiB', 'PiB', 'EiB'];
+        let i_tok = 0;
+        while (Math.abs(n) > 1024 && i_tok < tokens.length - 1) {
+            n /= 1024;
+            ++i_tok;
+        }
+        return n.toFixed(2) + " " + tokens[i_tok];
+    }
+
     const chart = flamegraph()
         .width(960)
         .setDetailsElement(document.getElementById("details"))
-        .onClick(d => history.pushState({ id: d.id }, "", `#${d.id}`));
+        .onClick(d => history.pushState({ id: d.id }, "", `#${d.id}`))
+{% if display_bytes %}
+        .setLabelHandler(function (d) {
+            return d.data.name + ' (' + d3.format('.3f')(100 * (d.x1 - d.x0), 3) + '%, ' + toHumanBytes(d.value) + ' )';
```

Comment:
This is modeled after the default: https://github.com/spiermar/d3-flame-graph/blob/c719193d7cb0a74a667ddd565d2717af0ac92429/src/flamegraph.js#L96